tests/js/spec/views/issueList/actions.spec.jsx

```diff
@@ -183,41 +205,151 @@ describe('IssueListActions', function() {
               datetime: {start: null, end: null, period: null, utc: true},
             }}
-            groupIds={[1, 2, 3, 6, 9]}
+            groupIds={['1', '2', '3', '6', '9']}
             onRealtimeChange={function() {}}
             onSelectStatsPeriod={function() {}}
             realtimeActive={false}
             statsPeriod="24h"
           />,
           TestStubs.routerContext()
         );
-      });
 
-      it('resolves selected items', function() {
-        const apiMock = MockApiClient.addMockResponse({
+        wrapper.setState({
+          allInQuerySelected: false,
+          anySelected: true,
+          multiSelected: true,
+        });
+
+        apiMock = MockApiClient.addMockResponse({
           url: '/organizations/1337/issues/',
           method: 'PUT',
         });
-        jest
+        storeMock = jest
           .spyOn(SelectedGroupStore, 'getSelectedIds')
-          .mockImplementation(() => new Set([3, 6, 9]));
+          .mockImplementation(() => new Set(['3', '6', '9']));
+      });
+
+      afterEach(function() {
+        MockApiClient.clearMockResponses();
+        storeMock.mockRestore();
+      });
 
-        wrapper.setState({allInQuerySelected: false, anySelected: true});
+      it('sets selected items to resolved', function() {
         wrapper
           .find('ResolveActions ActionLink')
           .first()
           .simulate('click');
+
         expect(apiMock).toHaveBeenCalledWith(
           expect.anything(),
           expect.objectContaining({
             query: {
-              id: [3, 6, 9],
+              id: ['3', '6', '9'],
               project: [1],
             },
             data: {status: 'resolved'},
           })
         );
       });
+
+      it('sets selected items to ignored', function() {
+        wrapper
+          .find('IgnoreActions ActionLink')
+          .first()
+          .simulate('click');
+
+        expect(apiMock).toHaveBeenCalledWith(
+          expect.anything(),
+          expect.objectContaining({
+            query: {
+              id: ['3', '6', '9'],
+              project: [1],
+            },
+            data: {status: 'ignored'},
+          })
+        );
+      });
+
+      it('merges selected items', function() {
+        wrapper
+          .find('ActionLink.action-merge')
+          .first()
+          .simulate('click');
+
+        // Let the modal open
+        wrapper.update();
+
+        wrapper
+          .find('ActionLink.action-merge Modal Button')
+          .at(1) // 0: "Cancel", 1: "Merge"
+          .simulate('click');
+
+        expect(apiMock).toHaveBeenCalledWith(
+          expect.anything(),
+          expect.objectContaining({
+            query: {
+              id: ['3', '6', '9'],
+              project: [1],
+            },
+            data: {merge: 1},
+          })
+        );
+      });
+
+      it('sets selected items to bookmarked', function() {
+        test.todo('Modal does not want to show');
```

Comment:
@billyvg I can't seem to get this modal to work.

Reply:
Halp.

Reply:
Give me a little bit and I'll try to help